tests/src/test/scala/cats/tests/EqSuite.scala

```diff
@@ -17,4 +17,27 @@ class EqSuite extends CatsSuite {
     checkAll("Eq[Int]", ContravariantMonoidalTests[Eq].contravariantMonoidal[Int, Int, Int])
     checkAll("ContravariantMonoidal[Eq]", SerializableTests.serializable(ContravariantMonoidal[Eq]))
 
+    case class Something(a: Int, b: Int, c: Int)
+
+    test("Eq returned by Eq.compose yields true on all underlying Eq's true") {
```

Comment:
Can we test the created instances against the `Eq` laws?

Reply:
Done